### ISSUE TITLE
Highlights on changes done between releases

### ISSUE THREAD

**Issue by fluxer** [eudev-project/eudev#107]
It would be nice to have some sort of notable changes done from release to release listed somewhere, either via NEWS, ChangeLog, whatever in the repository or at least on the releases page at github. This is just for convinience, for an example I've learned from other sources[1] that eudev v1.8 requires the input group. For packagers this means less time spend on scoping trogh the git log to look for possible blockers and adjustments they need to make for the package they maintain. Many projects provide this "feature", if I may call it such, and for me (as packages maintainer) it will mean a lot if you provide it.

Cheers!

[1] https://crux.nu/gitweb/?p=ports/core.git;a=commit;h=4d61a0d8f5b0abbc75e027c36913b0e8c29b494e

**Comment by blueness** [comment]
I hear you.  I have the luxury in gentoo that ssuominen follows systemd-udev and makes all those chnages for me.  I can add an important news item for distros like crux which need to know if they need new groups, if things are changing directories etc.

**Comment by blueness** [comment]
release 3.1 fixed --{enable,disable}-manpages.  manpages are now NOT installed by default if no option is given.  a not to package maintainers.

**Comment by blueness** [comment]
release 3.1 adds --{enable,disable}-hwdb which will optionally install the files in /etc/udev/hwdb.d.  The default is to install them since most people will want that.  --disable-hwdb is useful for embedded systems that want to save 5MB of space.

**Comment by blueness** [comment]
In the next release 3.2, libgudev has been broken out.  See https://wiki.gnome.org/Projects/libgudev.
Also gtk-docs has been dropped.

**Comment by udeved** [comment]
Do you know if systemd follows the libudev split with 230 @blueness ?
As it is atm, arch linux only have one bundled libsystemd blob, and it would be pretty messy, if this remains.

**Comment by udeved** [comment]
Forget about my question, I over read lib-g-udev, and read libudev.
libgudev has been split out.

**Comment by fbt** [comment]
@udeved if you want to have a split libsystemd package, look at libsystemd-standalone in AUR. It has all the components split into separate packages, including libsystemd-udev. I would also not mind feedback on the package :)

**Comment by udeved** [comment]
The trouble is, there is no libudev interface in the arch PKGBUILDs.

We can discuss it, I'll create an issue in package-openrc, so to keep this issue clean.

**Comment by blueness** [comment]
highlight for v3.2.1 - fixes build against glibc-2.23.  i'm testing now and will push the tagged release once the test is finished.

**Comment by bbonev** [comment]
Resolved with recent release model
